aksel.nav.no/website/pages/templates/404/complete.tsx

```diff
@@ -0,0 +1,138 @@
+import { BugIcon } from "@navikt/aksel-icons";
+import {
+  BodyShort,
+  Box,
+  Button,
+  HGrid,
+  Heading,
+  Link,
+  List,
+  VStack,
+} from "@navikt/ds-react";
+
+export default function Example() {
+  const styles = `
+  /* Må justeres basert på app inkludert header og footer  */
+  .app-container {
+    min-height: 80dvh;
+  }
+
+  /* Malen skal sentreres på siden med en maksbredde satt.  */
+  .app-container__layout {
+    max-width: 1024px;
+    margin-inline: auto;
+  }
+  `;
+
+  return (
+    <>
+      {/* Stiler bør trekkes ut i eget stylesheet, evt erstatte med tailwind */}
+      <style>{styles}</style>
+      {/* Header og Footer bør være inkludert i Mal */}
```

Comment:
```suggestion
      {/* Header og Footer bør være inkludert på siden */}
```
Med forbehold om at jeg kan ha misforstått kommentaren